diff --git a/dist/react/index.js b/dist/react/index.js
index e6ff156423b82514aa1a1f36929a55c034bdcfb1..a166e360f07bc7e400c2e1e8b1bbfb74f87e0dc2 100644
--- a/dist/react/index.js
+++ b/dist/react/index.js
@@ -8,7 +8,7 @@ import { jsx as _jsx } from "react/jsx-runtime";
 import { ConvexHttpClient } from "convex/browser";
 import { ConvexProviderWithAuth } from "convex/react";
 import { useContext, useMemo } from "react";
-import { AuthProvider, ConvexAuthActionsContext, ConvexAuthTokenContext, useAuth, } from "./client";
+import { AuthProvider, ConvexAuthActionsContext, ConvexAuthTokenContext, useAuth, } from "./client.js";
 /**
  * Use this hook to access the `signIn` and `signOut` methods:
  *
