diff --git a/dist/dts/Effect.d.ts b/dist/dts/Effect.d.ts
index 954e16106c3871ef91e55f77ca5db35c2a09d59a..8b6cc334e19136f4283e653110a0c36363c0bb23 100644
--- a/dist/dts/Effect.d.ts
+++ b/dist/dts/Effect.d.ts
@@ -525,10 +525,15 @@ export declare namespace All {
  * @since 2.0.0
  * @category collecting & elements
  */
-export declare const allSuccesses: <X extends Effect<any, any, any>>(elements: Iterable<X>, options?: {
-    readonly concurrency?: Concurrency | undefined;
-    readonly batching?: boolean | "inherit" | undefined;
-} | undefined) => Effect<Array<Effect.Success<X>>, never, Effect.Context<X>>;
+export declare const allSuccesses: <V, S>(
+    elements: Iterable<Effect<V,unknown,S>>,
+    options?:
+      | {
+        readonly concurrency?: Concurrency | undefined
+        readonly batching?: boolean | "inherit" | undefined
+      }
+      | undefined
+  ) => Effect<Array<V>, never, S>;
 /**
  * Drops all elements until the effectful predicate returns true.
  *
